docs/contribute.md

```diff
@@ -0,0 +1,55 @@
+# Contributor guide
+
+These sections describe how you can make contributions to this theme.
+
+
+## Run the tests
+
+The easiest way to run the test suite is using `nox`.
+This will install the local version of the package and run the test suite.
+
+```bash
+nox -s test
+```
+
+## Build the documentation
+
+The easiest way to build the documentation locally is using `nox`.
+To do so, run this command:
+
+```bash
+nox -s docs
+```
+
+To build the documentation with a liveserver that will automatically reload to show previews of your changes, use:
+
+```bash
+nox -s docs -- live
+```
+
+## Make a release
+
+The easiest way to make a release is to follow these steps:
+
+1. **Bump the version in `__init__.py` and push it to master**.
+   We try to use [semver version numbers](https://semver.org/) but don't stress out about it too much.
+   - In your commit message use something like `RLS: vX.X.X`.
```

Comment:
What level of contributor is this guide aimed at? Does `RLS: ` already make sense to them (I presume it's short for release)?